### PR TITLE
Bump operator sdk v1.7.2

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**

### CODE REVIEW

File: config/samples/jiraservicedesk_v1alpha1_project.yaml [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
-  key: STK
-  projectTypeKey: service_desk
-  projectTemplateKey: com.atlassian.servicedesk:itil-v2-service-desk-project
-  description: "Sample project for jira-service-desk-operator"
-  assigneeType: PROJECT_LEAD
-  leadAccountId: 5ebfbc3ead226b0ba46c3590
-  url: https://stakater.com
\ No newline at end of file
+  # Add fields here
+  foo: bar
```
Comment (abdulhaseeb2) [inline config/samples/jiraservicedesk_v1alpha1_project.yaml]: These changes should be reverted.

File: config/samples/jiraservicedesk_v1alpha1_customer.yaml [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
 spec:
-  name: testCustomer
-  email: testCustomer@test.com
-  projects:
-    - KEY1
-    - KEY2
-    - KEY3
\ No newline at end of file
+  # Add fields here
+  foo: bar
```
Comment (abdulhaseeb2) [inline config/samples/jiraservicedesk_v1alpha1_customer.yaml]: These changes should be reverted.

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/jira-service-desk-operator/commit/05626bd4abce331ac6adb6e8fa2cc012aaa03780/checks) has Failed!

File: config/manifests/bases/jira-service-desk-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -1,60 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
```
Comment (abdulhaseeb2) [inline config/manifests/bases/jira-service-desk-operator.clusterserviceversion.yaml]: should this file be deleted?

File: config/samples/jiraservicedesk_v1alpha1_customer.yaml (hunk 2 above)
Comment (abdulhaseeb2) [inline config/samples/jiraservicedesk_v1alpha1_customer.yaml]: changes should be reverted

File: config/samples/jiraservicedesk_v1alpha1_project.yaml (hunk 1 above)
Comment (abdulhaseeb2) [inline config/samples/jiraservicedesk_v1alpha1_project.yaml]: changes should be reverted

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

File: config/manifests/bases/jira-service-desk-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (ahmedwaleedmalik) [reply config/manifests/bases/jira-service-desk-operator.clusterserviceversion.yaml]: No, this needs to be re-generated based on the latest structure

File: config/samples/jiraservicedesk_v1alpha1_project.yaml (hunk 1 above)
Comment (ahmedwaleedmalik) [reply config/samples/jiraservicedesk_v1alpha1_project.yaml]: Good catch, let me update this.

File: config/samples/jiraservicedesk_v1alpha1_customer.yaml (hunk 2 above)
Comment (ahmedwaleedmalik) [reply config/samples/jiraservicedesk_v1alpha1_customer.yaml]: Good catch, let me update this.

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull stakater/jira-service-desk-operator:SNAPSHOT-PR-78-8ffeef49`

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull stakater/jira-service-desk-operator:SNAPSHOT-PR-78-f8ce6fbb`

**Review by abdulhaseeb2 — DISMISSED** [review]
LGTM

**Review by abdulhaseeb2 — APPROVED** [review]
LGTM

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull stakater/jira-service-desk-operator:SNAPSHOT-PR-78-b71958fb`

**Pull request merged**